modin/core/io/pickle/pickle_dispatcher.py

```diff
@@ -21,11 +21,7 @@
 
 
 class PickleExperimentalDispatcher(FileDispatcher):
-    """
-    Class handles utils for reading `.pkl` files.
-
-    Inherits some common from `FileDispatcher` class.
-    """
+    """Class handles utils for reading `.pkl` files."""
```

Comment:
```suggestion
    """Class handles utils for reading pickle files."""
```
I don't think `.pkl` is the standard extension, unlinke `.csv` or `.parquet`.